Code/Mantid/Framework/DataObjects/src/ReflectometryTransform.cpp

```diff
@@ -223,8 +229,27 @@ Mantid::API::MatrixWorkspace_sptr ReflectometryTransform::execute(
   return ws;
 }
 
+boost::optional<boost::shared_ptr<Mantid::DataObjects::TableWorkspace>>
+createTable(boost::optional<
+            boost::shared_ptr<Mantid::DataObjects::TableWorkspace>> &vertexes) {
+  (*vertexes)->addColumn("double", "Qx"); //needs to be set to whatever our Transform is.
+  (*vertexes)->addColumn("double", "Qy"); //needs to be set to whatever our Transform is.
+  (*vertexes)->addColumn("int", "OriginIndex");
+  (*vertexes)->addColumn("int", "OriginBin");
+  (*vertexes)->addColumn("double", "CellSignal");
+  (*vertexes)->addColumn("double", "CellError");
+
+  return vertexes;
```

Comment:
You are passing in the TableWorkspace as a reference argument, so there's absolutely no need to return a copy. This function should have a return type of void.